doc/2/getting-started/.flutter/lib/main.dart

```diff
@@ -0,0 +1,6 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_getting_started/login.dart';
+
+void main() {
+  runApp(Login());
+}
```

Comment:
To avoid futur unwanted git changes on this file
```suggestion
}

```